### DISCUSSION TITLE
How can I make a query to search for empty strings?

### DISCUSSION THREAD

**Discussion by akshaysasidrn** [PostgREST/postgrest#3035] [Q&A] [answered]
The question is quite trivial, but I'm not able to figure this out.
What should the API request be like to trigger a query equivalent to?
`SELECT * FROM your_table WHERE column_name = '';`

When I try to query like this:
` "GET /your_table?column_name=eq.'' HTTP/1.1" 200 - "" ""`

It searches for exact quotes within strings `"''"` rather than for empty string. Please do advise.

**Comment by steve-chavez** [comment, marked as answer]
Try it like:

```bash
curl 'localhost:3000/projects?name=eq.'
[{"id":333,"name":"","client_id":null}]
```

I can confirm that it works and finds the empty string.

**Reply by akshaysasidrn** [reply]
Thanks, that works!
